### PR TITLE
Update docs and add new aliases for unpack and eject

### PR DESCRIPTION
**Pull request by stevie-mayhew**

### CODE REVIEW

**Comment by stevie-mayhew** [comment]
I'm also wondering if we need both `RequireRecipeCommand` and `UpdateRecipeCommand`

They do the same thing except one errors when the recipe doesn't exist, and I don't think we need to do that necessarily?

File: src/RequireRecipeCommand.php [hunk 1]
```diff
@@ -19,6 +19,7 @@ public function configure()
     {
         $this->setName('require-recipe');
         $this->setDescription('Invoke this command to inline a recipe into your root composer.json');
+        $this->setAliases(['eject', 'unpack']);
```
Comment (Cheddam) [inline src/RequireRecipeCommand.php:22]: Could you give some context on why this needs two different aliases?

Comment (stevie-mayhew) [reply src/RequireRecipeCommand.php:22]: I believe that this was due to the naming convention used in other projects which do the same, i.e. create-react-app.

To be honest I can no longer remember

**Comment by GuySartorelli** [comment]
Closing this PR. I don't think adding aliases adds any value here - if anything it increases ambiguity. That and the fact that Stevie has said they don't remember what the aliases are for lead me to think this isn't a change we should be merging in.

Feel free to reopen and discuss if you disagree.

**Pull request closed without merging**